lib/RAST_SDK/RAST_SDKImpl.pm

```diff
@@ -1540,18 +1540,26 @@ sub annotate_genomes
 		}
 	}
 		
-		my $output_genomeset;
-		if (defined $params->{output_genome} && $params->{output_genome} gt ' ') { 
-			my $output_genomeset = $params->{output_genome};
-	        my $genome_set_name = $params->{output_genome};
-	        my $genome_set = Bio::KBase::kbaseenv::su_client()->KButil_Build_GenomeSet({
-	            workspace_name => $params->{workspace},
-	            input_refs => $output_genomes,
-	            output_name => $output_genomeset,
-	            desc => 'GenomeSet Description'
-	        });
-		}
-
+    my $output_genomeset;
+    if (defined $params->{output_genome} && $params->{output_genome} gt ' ') {
+        my $output_genomeset = $params->{output_genome};
+        my $report_reference = Bio::KBase::kbaseenv::su_client()->KButil_Build_GenomeSet({
+            workspace_name => $params->{workspace},
+            input_refs => $output_genomes,
+            output_name => $output_genomeset,
+            desc => 'GenomeSet Description'
+        });
+
+        my $newset = Bio::KBase::kbaseenv::get_objects([{
```

Comment:
Ideally, I'd use get_object_info3 or use the `no_data` option but I suppose genome sets are pretty small